src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { User } from '../User';
+
+export const Todo = ({ todo }) => (
+  <>
+    <td className="table-secondary">
+      {todo.id}
+    </td>
+
+    <td className="table-light">
+      {todo.title}
+    </td>
+
+    <td className="table-secondary">
+      {todo.completed
+        ? 'true'
+        : 'false'
```

Comment:
```suggestion
        ? 'true'
        : 'false'
```

думаю будет лучше назвать состояния completed/active либо что-то такое